### DISCUSSION TITLE
Getting - Something went wrong. Please check your network settings

### DISCUSSION THREAD

**Discussion by Kams353** [microsoft/MicrosoftEdge-Extensions#191] [Q&A]
When trying to upload new version of package receiving - Something went wrong. Please check your network settings message. 

![image](https://github.com/user-attachments/assets/9f269a4e-a097-49d7-9d3a-9a455fa858c7)

**Comment by mikemarquez** [comment]
I am seeing the same issue. :-( Is there some kind of status page I can check?

**Comment by ManikanthMSFT** [comment]
Hey @Kams353, @mikemarquez, we wanted to inform you that our team has successfully addressed the issue you reported. Could you kindly verify on your end and confirm if everything is working as expected while uploading/updating your extension?

Please don't hesitate to reach out if you experience any further issues or require additional assistance.

**Reply by mikemarquez** [reply]
Thank you for the follow-up.  Everything seems fine now on my end.  I am still curious as to whether there is a status page that could have alerted us to the issue.

**Reply by ManikanthMSFT** [reply]
Hey @mikemarquez, thank you for your confirmation and your valuable feedback! We’ve taken note of your suggestion and will be forwarding it to our team for further consideration.

If you have any other ideas or need assistance, feel free to reach out.
